### DISCUSSION TITLE
Compilation error all the time for NewPipeExtractor:NewPipeExtractor.

### DISCUSSION THREAD

**Discussion by NanyangTaiji** [TeamNewPipe/NewPipe#10627] [Questions]
Could not find com.github.TeamNewPipe.NewPipeExtractor:extractor:289db1178ab66694c23893e6a487d4708343c47b.
Required by:
    project :app > com.github.TeamNewPipe:NewPipeExtractor:289db1178ab66694c23893e6a487d4708343c47b
Search in build.gradle files

Staring from 'com.github.TeamNewPipe.NewPipeExtractor:NewPipeExtractor:v0.22.2', none of the extractor available.

**Comment by deep-soft** [comment]
insert in ci.yml (https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/workflows/ci.yml)
after checkout (      - uses: actions/checkout@v3), line 39

sed -ibak "s!NewPipeExtractor:289db1178ab66694c23893e6a487d4708343c47b!NewPipeExtractor:289db117!" app/build.gradle;

Reason:
not found
com.github.TeamNewPipe.NewPipeExtractor:extractor:289db1178ab66694c23893e6a487d4708343c47b
but exist
com.github.TeamNewPipe.NewPipeExtractor:extractor:289db117
and compilation is succesfull with this.
